changelogs/OUTCOME_ATH_DICTIONARY_CHANGELOG.md

```diff
@@ -12,32 +12,50 @@ We define here in what versions of the data dictionaries contains which tables
 
 - *non-repeated variables*
   - Added variables see list below
+- *monthly-repeated variables*
+  - Added variables see list below
 - *trimester-repeated variables*
   - Added variables see list below
+- *yearly-repeated variables*
+  - Added variables see list below
 
 #### Additional variables
 
-- see lists below
+- none, this is the first release
 
 *non-repeated variables*
 
 **Added variables**
 
 | Variable        | Type        | Description                                      |
 | ----------------| ----------- | --------------------------------------------- |
-| row_id          | integer     | Unique identifer for the row in Opal          |
```

Comment:
Not specific for Opal right? So 'Unique identifier for the row' or 'Unique identifier for the row in Opal and Armadillo'

Reply:
It might actually be Opal specific, not sure. Armadillo does not need them and dsUpload will create row_id's on runtime so you do not need to define them in your data.